Examples/TestServerClientPackets/TestServerClientPackets.csproj

```diff
@@ -1,4 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup>

```

Comment:
no clue that this happened. no clue *what* happened. you wil need to explain this more

Reply:
A System.ValueTuple NuGet has been added as a Dependency. Which is ok, since this is the Example proj. My comment is related to the TestServer and TestClient. both should have an identical .csproj, which isn't the case.